phpBB/docs/CREDITS.txt

```diff
@@ -26,6 +26,7 @@ phpBB Lead Developer:    Marc (Marc Alexander)
 
 phpBB Developers:        bantu (Andreas Fischer)
                          CHItA (Máté Bartus)
+                         Derky (Derk R.)
                          Elsensee (Oliver Schramm)
                          Nicofuma (Tristan Darricau)
                          prototech (Cesar Gallegos)
```

Comment:
Remove Cesar and add Michael while you're on it.

Reply:
Also, I think hanakin should also be added here.

Reply:
Micheal should have his own welcome ticket.